src/migrations/data.ts

```diff
@@ -343,6 +385,55 @@ function dropCommands(initialMessages, commandType: string) {
   return messages;
 }
 
+function fixUpdateChart(data: Partial<WorkbookData>, initialMessages: StateUpdateMessage[]) {
```

Comment:
I don't understand why it's needed since `cmd.sheetId` is unused anyway
(and should be eventually removed)

Reply:
again, used in 16.0+ atm